cmip6_downscaling/methods/bcsd/tasks.py

```diff
@@ -183,7 +183,7 @@ def postprocess_bcsd(
     title = "bcsd_postprocess"
 
     ds_hash = str_to_hash(str(bias_corrected_fine_full_time_path) + str(spatial_anomalies_path))
-    target = intermediate_dir / title / ds_hash
+    target = results_dir / title / ds_hash
```

Comment:
@norlandrhagen - pointing this out because it impacts your work on the bcsd flow.